text/0685-new-browser-support-policy.md

```diff
@@ -0,0 +1,293 @@
+to was released. In other words, no breakage can or should occur due to new
+browser feature usage in _any_ patch release, and if it does, it is a bug.
+
+For bug reports, support is determined by combining our existing Ember version
+support policy with the browser versions that were supported at the time an
+Ember version was released. This means Ember will work as time and resourcing is
+available to fix any browser specific issues that occur in:
+
+1. The current stable and LTS releases of Ember and Ember Data
+2. For browsers that were supported by those versions when they were relased
+
+For the previous LTS release, only security bugfixes will be supported,
+following the existing LTS policy.
+
+## Future changes to the policy
+
+This policy is not meant to be immutable. Over time, new browsers could be added
+to the support matrix, and explicit exceptions could be added for "critical
+internet infrastructure".
+
+For example, it was well known that Google's search crawler used a very old
+version of Google Chrome for a very long time. It no longer does and is
+regularly updated, but during that time frame it was important to support that
+version of Chrome. Likewise, it was very important to support IE11 for a very
+long time since it had a large usage percentage, especially in corporate
+environments. While none of these cases are known to exist at the time of this
+writing, if one should arise it may be added explicitly via RFC.
+
+Future RFCs may amend this policy in a strictly additive way without requiring
+a major version bump in Ember. Newly supported browsers will begin being
+supported in the next minor version of Ember after the updated support policy is
+implemented.
+
+Future RFCs that amend this policy to _remove_ support for a browser will
+_require_ a major version of Ember to implement.
+
+## Implementation
+
+This policy drops support for a major browser, and therefore can only be
+implemented with a major version bump.
+
+## How we teach this
+
+- A blog post explaining the policy with a link to this RFC will be posted.
+- Browser support policy will be linked in repo
+- Browser support policy will be added to the website
+- Browser support policy will have an announcement in the Ember Times
+
+## Drawbacks
+
+- The proposed policy makes Safari the new "lowest common denominator" browser,
+  replacing IE11. Since the supported versions of Safari will continue to be
+  supported indefinitely, we will not be able to use any new browser features
+  added afterwards without another major version. In particular, the spec for
+  `WeakRef` has finally been stabilized, and this is something we'll likely want
+  to make use of in the not-to-distant future.
+
+  While this is true, it does not impact the immediate roadmap for Ember in the
+  next few years. When the time comes, we can update our browser support policy
+  again, and release a new major version.
+
+## Alternatives
```

Comment:
* Drop IE11 but not add more definition to support (keeping it as a case by case determination by the core teams)